optaplanner-core/pom.xml

```diff
@@ -134,24 +136,6 @@
 
   <build>
     <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
Why do you remove this?

Reply:
Because the overrides are no longer necessary. The build now passes without it.
The dep plugin still runs, because that is defined elsewhere. If that is not the case then, obviously, that is wrong.

Reply:
You're right. Thanks for catching this. It's always nice to see a piece of code to go away because it turns out to be doing nothing :slightly_smiling_face: